codesandbox/examples/cra/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is the CRA template, right? Should the dir be /templates not /examples?

Reply:
Yeah good point! I'll change it to templates, much more descriptive!

Another idea I had is that we could also save the react version in gatsby config, then you only need to change the react version in one file to update all templates.